packages/wallet/src/unipig-transitioner.ts

```diff
@@ -87,6 +90,26 @@ export class UnipigTransitioner extends DefaultWallet {
     this.db = db
     this.stateSolver = stateSolver
     this.knownState = {}
+    this.wallet = wallet
+  }
+
+  public async sign(signer: string, message: string): Promise<string> {
+    if (typeof this.wallet !== 'undefined') {
+      log.debug('Were signing with ethers!')
```

Comment:
Lol tbh I meant to remove this because really I was using it as a quick debugging test. That said, I updated the log message to be a bit more descriptive and kept it in. Feel free to delete though if you think it's too noisey.